website/blog/2019-10-28-react-native-doctor.md

```diff
@@ -0,0 +1,60 @@
+---
+title: Meet Doctor, a new React Native command
+author: Lucas Bento
+authorTitle: React Native Community
+authorURL: https://twitter.com/lbentosilva
+authorImageURL: https://avatars3.githubusercontent.com/u/6207220?s=460&v=4
+authorTwitter: lbentosilva
+category: announcements
+---
+
+After 160+ commits from 6 contributors, the React Native Community is excited to launch `doctor`. It's a brand-new React Native CLI command to help you out with getting started, troubleshooting and automatically fixing errors with your development environment. `doctor` is heavily inspired by [Expo](https://expo.io/) and [Homebrew](https://brew.sh/)'s own doctor command with a pinch of UI inspired by [Jest](https://jestjs.io/).
+
+Here it is in action:
+
+<p style="text-align: center;">
+  <video width="700" controls="controls" autoplay style="border-radius: 5px;">
+    <source type="video/mp4" src="/react-native/img/homepage/DoctorCommand.mp4"></source>
+  </video>
+</p>
+
+## How it works
+
+The `doctor` command currently supports most of the software and libraries that React Native relies on, such as CocoaPods, Xcode and Android SDK. With `doctor` we'll find issues with your development environment and give you the option to automatically fix them. If `doctor` is not able to fix an issue, it will display a message and a helpful link explaining how to fix it manually as the following:
+
+<p style="text-align: center;">
+  <img width="700" src="/react-native/img/DoctorManualInstallationMessage.png" alt="Doctor command with a link to help on Android SDK's installation" title="Doctor command with a link to help on Android SDK's installation" />
+</p>
+
+## Try it now
+
+The `doctor` command is available as a part of React Native 0.62. However, you can try it without upgrading yet:
+
+```sh
+npx @react-native-community/cli@3 doctor
```

Comment:
v3 is now tagged as `latest`
```suggestion
npx @react-native-community/cli doctor
```

Reply:
😫 I'm gonna change this and the video.

It should work with no tag, right?

Reply:
It will also work with the tag, don't worry :) I wouldn't change the video

Reply:
I think I already have a video without the tag so I'll try to find it.